tmt/steps/provision/local.py

```diff
@@ -54,12 +54,15 @@ class GuestLocal(tmt.Guest):
     def ansible(self, playbook, extra_args=None):
         """ Prepare localhost using ansible playbook """
         playbook = self._ansible_playbook_path(playbook)
+        self.run(f'sudo sh -c "stty cols {tmt.utils.OUTPUT_WIDTH};:"',
+                 shell=True)
         stdout, stderr = self.run(
-            f'sudo sh -c "stty cols {tmt.utils.OUTPUT_WIDTH}; '
-            f'{self._export_environment()}ansible-playbook '
-            f'{self._ansible_verbosity()} '
-            f'{self._ansible_extra_args(extra_args)} -c local -i localhost,'
-            f' {playbook}"')
+            ('sudo -E '
+             'ansible-playbook '
+             f'{self._ansible_verbosity()} '
+             f'{self._ansible_extra_args(extra_args)} -c local -i localhost, '
+             + playbook).split(),
```

Comment:
Let's not use the `split()` method here. It could possibly still be dangerous to auto-split the `playbook` content.